cmake/PythonApplyPatches.cmake

```diff
@@ -21,7 +21,12 @@ function(_apply_patches _subdir)
   foreach(patch IN LISTS _patches)
     set(msg "Applying '${patch}'")
     message(STATUS "${msg}")
-    set(applied ${PROJECT_BINARY_DIR}/CMakeFiles/patches/${patch}.applied)
+    set(applied ${SRC_DIR}/.patches/${patch}.applied)
```

Comment:
How is this directory supposed to be created? It doesn't exist on my system, but patches in `${PROJECT_BINARY_DIR}/CMakeFiles/patches/${patch}.applied)/*` does exist now.

Reply:
ah nevermind, I see this variable is used later in the post-patch step.